### DISCUSSION TITLE
Notices published under Directive 23

### DISCUSSION THREAD

**Discussion by maximAtanasov** [OP-TED/ted-xml-data-converter#15] [Q&A] [answered]
Hello there, thank you for your work on the converter!

I have a question regarding the limited scope of the converter. The current description mentions that notices published under directive 23 are not supported, however the conversion seems to work perfectly fine to me (converting an F05 notice). Could you please clarify what is meant by this?

**Comment by pdonohoe** [comment, marked as answer]
Thank you Maxim for your question, and apologies for the long delay in responding.  This is a typo, it should say that notices published under Directive 81 (the Defence forms) are not supported.
The typo will be corrected with the next release of the converter, which will be published in the next few days.
We have started work on the PIN Defence form (16), and this will be included in the following release.
